moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -29,18 +29,33 @@ const getHttpMethod = knackProjectId => {
   return knackProjectId ?? false ? "PUT" : "POST";
 };
 
+/**
+ * Function to map the signal IDs from a project object into an array and return the array length.
+ * @returns integer
+ */
+const countSignalsInProject = project => {
+  const signalIds = project.moped_proj_features.map(
+    feature => feature.location.properties.signal_id
```

Comment:
@frankhereford PR #553 is going to break this line (L38). the fix will be to change this:
```
feature => feature.location.properties.signal_id
```
to 
```
feature => feature.feature.properties.signal_id
```

i don't foresee any conflicts—so we can decide how to orchestrate the change after both of our branches are stable

Reply:
@frankhereford i was wrong all points—let's sync when both our branches are stable 🤪

Reply:
Sounds good - looking forward to it! 🙃